Phone bottom bar and some styling

chuyiting opened the pull request:
The bottom bar does not work with the ace editor. It should disappear when you scroll down, anyone can help solve this question??

chuyiting committed:
Phone bottom bar and some styling

chuyiting committed:
Repl prototype(The runner is buggy)

Will work with module team soon to solve the problem.

chuyiting committed:
Run botton works

Library loader has bug, will solve it soon with module team, and the phone bottom bar has to be solved, currently, the global state is cleaned up during the routing

benclmnt commented:
hi, i don't think it is necessary to change global state just to incorporate styling
We can work out some kind of carousel if you want, that way is nicer than changing routes imo.

see #16 .

wdyt?

benclmnt commented on public/index.html:
```diff
@@ -24,6 +24,16 @@
+    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
+    <script type="text/javascript">
+      var zero = 0;
+      $(document).ready(function() {
```
what is this for?
why add jquery? you can just put your javascript code in some hooks right?

benclmnt reviewed (dismissed):
May i know why you literally copy Workspace to WorkspaceCom?

I am suggesting of using a carousel that would not affect how state is managed right now in the computer view, so that we dont need to handle states based on device view port.

Other than that, there is just some minor conflicts that needs to be resolved when merging.

wdyt?

chuyiting committed:
Library, Runner, error message are all resolved

Some detailed styling  include the separating lines for repl and the flexible size of output still need to work on

benclmnt commented on tsconfig.json:
```diff
@@ -1,11 +1,7 @@
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "es2016",
```
any reason as to change this to es2016?

benclmnt commented:
pls resolve the conflicts so that i can merge it with the develop branch. thx

chuyiting committed:
Update index.html

chuyiting committed:
Update tsconfig.json

chuyiting committed:
Merge branch 'REPL-handling' of https://github.com/source-academy/journey-frontend into REPL-handling

chuyiting commented:
> May i know why you literally copy Workspace to WorkspaceCom?
> 
> I am suggesting of using a carousel that would not affect how state is managed right now in the computer view, so that we dont need to handle states based on device view port.
> 
> Other than that, there is just some minor conflicts that needs to be resolved when merging.
> 
> wdyt?

benclmnt committed:
Update index.html

erase </script> dangling

chuyiting committed:
Update localStorage.ts

chuyiting committed:
Merge branch 'REPL-handling' of https://github.com/source-academy/journey-frontend into REPL-handling

chuyiting committed:
Changes for merges
